README.md

```diff
@@ -1 +1,57 @@
-# 500apps-public-gtm-ninjachat
\ No newline at end of file
+# Ninjachat by 500apps
```

Comment:
can we say 500apps Ninjachat

So that it comes as the first app in the GTM

Reply:
Modified Sir